tests/testExecutables.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+import lsst.utils.tests
+
+
+class UtilsBinaryTester(lsst.utils.tests.ExecutablesTestCase):
+    pass
+
+EXECUTABLES = ['testPsfAttributes', 'testSizeMagnitudeStarSelector', 'testImagePsf', 'testPsfCaching', 'testWarpedPsf']
```

Comment:
It will auto detect. If you give a list then it will report if the test executable can't be found (maybe it wasn't built). If you don't give a list it will only test what it can find.